plane/fit_plane.py

```diff
@@ -53,18 +53,22 @@ def from_template_centers(cls, template_center_positions_uv_pix, template_center
         return fp
         
     @classmethod
-    def from_json(cls, json_str):
+    def _from_json(self, json_str):
```

Comment:
Can you explain why you rewrote from_json?
It seems that the original function is better (more documentation).

Reply:
Sorry, deleted the wrong thing. fixed